docs/source/common/trainer.rst

```diff
@@ -241,6 +241,24 @@ Example::
 
     Trainer(accelerator=MyOwnAcc())
 
+.. note::
+    If the `devices` flag is not defined, it will assume `devices`` to be `"auto"` and fetch the `auto_device_count`
+    from the accelerator.
```

Comment:
```suggestion
.. note::

    If the ``devices`` flag is not defined, it will assume ``devices``` to be ``"auto"`` and fetch the ``auto_device_count``
    from the accelerator.
```